functionary/inference_stream.py

```diff
@@ -5,13 +5,19 @@
 import torch
 from transformers import LlamaForCausalLM, LlamaTokenizer
 from transformers.generation.logits_process import (
-    LogitsProcessorList, RepetitionPenaltyLogitsProcessor,
-    TemperatureLogitsWarper, TopKLogitsWarper, TopPLogitsWarper)
+    LogitsProcessorList,
```

Comment:
The same here, server.py also uses this file so I am not sure we should put thing from vllm to this file